packages/itwin/property-grid/package.json

```diff
@@ -59,6 +59,7 @@
     "@itwin/core-react": "^3.0.0",
     "@itwin/eslint-plugin": "^3.0.0",
     "@itwin/imodel-components-react": "^3.0.0",
+    "@itwin/itwinui-react": "^1.32.0",
```

Comment:
no lockfile changes?

Reply:
Sorry, wdym? Is there supposed to be a lockfile? I just did `pnpm install` in my cospace.

Reply:
oh youre doing it in a cospace,

so this looks good, but youll have to open up this branch in another clone of it on your machine to do the rush update and rush change